libraries/java11/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Pull base image.
+
+RUN apt-get -y update && \
+    apt-get install --no-install-recommends -y \
+        software-properties-common \
+        build-essential \
+        ca-certificates \
+        libssl-dev \
+        apt-transport-https \
+        wget \
+        default-jdk \
+        maven && \
+	add-apt-repository ppa:openjdk-r/ppa && \
+	apt-get update && \
+	apt-get install -y \
+	openjdk-11-jdk && \
```

Comment:
this can result in somewhat inconsistent builds since the version of openjdk-11 might change. is it possible to add a version here like `=11.x.x`?  that way every build will be consistent not matter when it is run, and if we want to bump minor versions we'd just edit this file (which results in a different commit in langpacks and future image builds will have that as a label).

i'd try to use a reference dockerfile if possible for someone else who's built reproducible jdk images maybe from https://docs.docker.com/samples/library/openjdk/ (including licenses and stuff)

Reply:
I'll investigate